test/fpm_test/test_package_dependencies.f90

```diff
@@ -221,6 +221,8 @@ subroutine resolve_dependency_once(self, dependency, root, error)
         character(len=*), intent(in) :: root
         !> Error handling
         type(error_t), allocatable, intent(out) :: error
+        !> Name of the parent package
+        character(len=*), intent(in), optional :: parent
```

Comment:
Is this used?

Reply:
It is not used, but it fixed the following error:
```
test/fpm_test/test_package_dependencies.f90:16:17:

procedure :: resolve_dependency => resolve_dependency_once
        1
Error: ‘resolve_dependency’ at (1) must have the same number of formal arguments as the overridden procedure
```